index.html

```diff
@@ -30,37 +31,61 @@ <h1 class="detail">I’m a software developer</h1>
         <li class="logos"><a href="#"><img src="./src/angellist.png" alt="angellist"></a></li>
         </ul>
     </section>
+
     <section class="header-layout">
-        <button class="scroll-down"></button>
+        <form action=#my-projects><button class="scroll-down"></button></form>
         <h1 class="recent">My Recent<br> Works</h1>
         <img src="./src/Indicator.png" alt="seperator">
     </section>
```

Comment:
- [ ] It would have been better if you included this section inside the section tag with class `layout` to make your code more uniform.

Reply:
I included it there thank you for your feedback